guides/micronaut-java-records/micronaut-java-records.adoc

```diff
@@ -171,8 +173,6 @@ curl http://localhost:8080/books
 
 common:graal-with-plugins.adoc[]
 
-WARNING: Due to a https://github.com/oracle/graal/issues/3984[bug with GraalVM and Java Records] it is necessary to include the flag `--report-unsupported-elements-at-runtime` when building the native executable. Create the file `native-image.properties`:
```

Comment:
If you removed the properties file, shouldn't you have removed the next line too?